src/Hint/All.hs

```diff
@@ -75,10 +76,20 @@ builtin x = case x of
 builtinHints :: [(String, Hint)]
 builtinHints = [(drop 4 $ show h, builtin h) | h <- [minBound .. maxBound]]
 
+-- Toggle implementation.
+#define HLINT_GHC_MATCH_ENABLE 0
 -- | Transform a list of 'HintBuiltin' or 'HintRule' into a 'Hint'.
```

Comment:
Do we need CPP here? Why not just if True then ... else ...

Reply:
Because stupid.